### PR TITLE
updating to latest KeyVault package.

### PR DESCRIPTION
**Pull request by ryuyu**
Add new settings to configs for keyvault.

@skofman1 @scottbommarito @maartenba

### CODE REVIEW

**Comment by scottbommarito** [comment]
LGTM

File: NuGet.Services.Dashboard.FrontEnd/Web.config [hunk 1]
```diff
@@ -31,7 +31,9 @@
     <add key="keyVault-VaultName" value="" />
     <add key="keyVault-ClientId" value="" />
     <add key="keyVault-CertificateThumbprint" value="" />
-    <add key="keyVault-ValidateCertificate" value="false"/>
+    <add key="keyVault-ValidateCertificate" value="false" />
+    <add key="keyVault-StoreName" value="" />
+    <add key="keyVault-StoreLocation" value="" />
```
Comment (skofman1) [inline NuGet.Services.Dashboard.FrontEnd/Web.config]: add default values (My, CurrentUser)

**Comment by skofman1** [comment]
:shipit:

**Comment by maartenba** [comment]
I'd indeed add default values here: https://github.com/NuGet/NuGet.Services.Dashboard/pull/191/files/86905a72c4fb82013c7494f41cc3635ed6ebb79b#r73057644

Other than that LGTM, :ship:

**Pull request merged**